app/src/main/java/com/imashnake/animite/features/searchbar/SearchFrontDrop.kt

```diff
@@ -145,9 +181,7 @@ fun ExpandedSearchBar(viewModel: SearchViewModel = viewModel()) {
             value = text,
             onValueChange = { input ->
                 text = input
-                viewModel.run {
-                    searchAnime(input)
-                }
+                viewModel.searchAnime(input)
```

Comment:
ty for removing `viewmodel.run { }` that's gross

Reply:
I could swear this was in the Google's architecture docs before I knew how scope functions worked.